spec/models/artist_spec.rb

```diff
@@ -268,6 +268,29 @@
     end
   end
 
+  describe 'remind_for_open_studios_register?' do
```

Comment:
Along with the other comments, these tests could move to the `UserNavigation` specs and the `Artist` model shouldn't need to change at all.

and they'll become something like

```
expect(user_nav.remind_for_open_studio_register?).to eq(true)
```

with basically the same setup.

I suppose we might want an extra test for when there is no current open studios event (when `OpenStudiosEvent.current == nil`) which can happen (but it's rare)

Reply:
Yea I thought about the `nil` situation, but I don't think it would be at this point. This goes back to my previous comment about how current open studios event is set up.

Reply:
yep.  it shouldn't ever be nil so who cares.  nevermind about the extra test.